File: java/sourceafis/src/sourceafis/general/SizeF.java
package sourceafis.general;

public class SizeF {
	public float Width;
	public float Height;

	public SizeF(float width, float height) {
		Width = width;
		Height = height;
	}

	public SizeF(PointF point) {
		Width = point.X;
		Height = point.Y;
	}
}

File: SourceAFIS/General/SizeF.cs
using System;

namespace SourceAFIS.General
{
    public struct SizeF
    {
        public double Width;
        public double Height;

        public SizeF(double width, double height)
        {
            Width = width;
            Height = height;
        }

        public SizeF(PointF point)
        {
            Width = point.X;
            Height = point.Y;
        }
    }
}
